packages/common/src/ui/components/inputs/Dropdown/Dropdown.tsx

```diff
@@ -1,16 +1,92 @@
 import React, { FC } from 'react';
-import Select, { SelectProps } from '@material-ui/core/Select';
-import MenuItem, { MenuItemProps } from '@material-ui/core/MenuItem';
+import {
+  MenuItem,
+  MenuItemProps,
+  FormControl,
+  Select,
+  InputLabel,
+  outlinedInputClasses,
+  Box,
+  SvgIconProps,
+  ListItemText,
+} from '@material-ui/core';
 import { styled } from '@material-ui/system';
 
-export const DropdownItem: FC<MenuItemProps> = props => {
-  return <MenuItem {...props} />;
+import { ChevronDown } from '../../../icons';
+
+interface DropdownItemProps extends MenuItemProps {
+  IconComponent?: React.JSXElementConstructor<SvgIconProps>;
+  inset?: boolean;
+}
+
+export const DropdownItem: FC<DropdownItemProps> = props => {
+  return (
+    <MenuItem sx={{ fontSize: '10' }} {...props}>
+      {props.IconComponent ? (
+        <Box mr={1}>
+          <props.IconComponent fontSize="inherit" />
+        </Box>
+      ) : null}
+      <ListItemText
+        inset={props.inset}
+        primaryTypographyProps={{ sx: { fontSize: 'inherit' } }}
+        sx={{ '&.MuiListItemText-inset': { paddingLeft: '22px' } }}
+      >
+        {props.children}
+      </ListItemText>
+    </MenuItem>
+  );
```

Comment:
A little bit of extras in here. I only noticed the icons in the zeplin a little later

![image](https://user-images.githubusercontent.com/35858975/132933963-5ed945db-0e25-4678-bd9a-a7052e910ae1.png)

And then added a little bit to be able to inset the font and have a nicer mix of rows with icons and those without. 

It's a little restrictive in the font sizes.. somewhat intentionally (for consistency) and somewhat because I didn't want to spend anymore time on this..!

Reply:
I hadn't noticed the icons before, though that looks like one of the older designs 🤷‍♂️